doc/examples/onboarding.example.yaml

```diff
@@ -50,7 +50,9 @@ addon:
 ## Add-on deployment instructions
   usageInstructions: >
     Follow  instructions in: https://docs.example.com/instructions
-## List of add-on supported kubernetes versions
+  deliveryOptionDescription: >
```

Comment:
Could you add additional documentation on what this means for anyone writing the file? What part of the upstream Marketplace API is the correlating to?